### ISSUE TITLE
Viewing annotations strips down margin in pattern

### ISSUE THREAD

**Issue by nielsvanmidden** [pattern-lab/patternlab-node#215]
## Background

Annotations are stripping down pattern margins.
### Steps to reproduce
- Open up an organism
- Show annotations
- Close annotations

As you can see in the images attached, the margins are now stripped down.

![3](https://cloud.githubusercontent.com/assets/6792036/11952439/9acaaba8-a898-11e5-8ce9-3cc7c27d4997.png)
![2](https://cloud.githubusercontent.com/assets/6792036/11952441/9acbcf2e-a898-11e5-8447-af43493d0999.png)
![1](https://cloud.githubusercontent.com/assets/6792036/11952440/9acaca98-a898-11e5-925a-08e03792ce10.png)

**Comment by bmuenzenmeyer** [comment]
Hi @nielsvanmidden - thanks for reporting and for using Pattern Lab. I quick tried this with some existing annotations I have in a 0.14.0 build of PL Node and did not witness this behavior.

I will set up a heading block like this and try again. Any additional mustache, css, or annotation logic you can provide would be helpful.

**A small final note** I'll be moving toward a more unified experience with [PL PHP starterkits](https://github.com/pattern-lab/styleguidekit-assets-default/tree/dev) someday, so substantial improvements to the PL Node frontend will likely not occur via me.

**Comment by nielsvanmidden** [comment]
Ah, so this means it is better to report general front end issues here?

I'm using PL-node-v1.0.0.

**Comment by bmuenzenmeyer** [comment]
Frontend issues can certainly be reported here. The point I was trying to make, _ineloquently_ at that, is that it's tougher for me to pinpoint exactly where they may get prioritized, since some are duplicates of things found in PL PHP, some are unique, some are interesting/quirky (like yours), and some are going to be blown away by proposed/future changes to the frontend.

tl;dr: Open a ticket and I can play clean up.

Thanks!

**Comment by nielsvanmidden** [comment]
Ah, thanks for the explanation.

**Comment by nielsvanmidden** [comment]
Will the assets in the [style-guide-assets repository](https://github.com/pattern-lab/styleguidekit-assets-default/tree/dev) eventually be a dependency of PL Node and PL PHP?
And does it make sense to report certain issues in that repository then?

In the attachment you can find my patterns, annotations and style.
[pattern-demo.zip](https://github.com/pattern-lab/patternlab-node/files/81424/pattern-demo.zip)

**Comment by bmuenzenmeyer** [comment]
Hi @nielsvanmidden that is my eventual goal, yes. I can't speak to how long it will be until I get there. 
That repo is a dependency for PL PHP v2 as I recall.
